### PR TITLE
Chore: IRD-2043 update documentation

### PR DESCRIPTION
**Pull request by MPIngeno**
- Updated README
- Applied a small change from a remaining PR comment

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,11 +21,71 @@ and reusable workflows.
...
-Once the release is created, update the version used within the repository where needed. See the [GitHub examples using versioned actions](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-versioned-actions).
+### Release Philosophy
+
+#### Workflow Initiation
+To ensure accuracy throughout the release cycle and to avoid unnecessary version bumps,
+releases are initiated manually.
+
+#### Versioning
+A unified versioning system is implemented for all actions, ensuring uniformity across the entire project.
+This approach uses Lerna, which automatically adjusts the version of all actions and workflows in the monorepo.
```
Comment (jonathanpelletier) [inline README.md:40]: Not sure the term "monorepo" is accurate for here 🤔

File: README.md [hunk 2]
```diff
@@ -21,11 +21,71 @@ and reusable workflows.
...
+When a release is initiated, a Pull Request is created with the changelog and updated version. After this PR is
+approved and merged, the associated workflow will generate a release note and create a release with the proper tag.
+The unified versioning ensures a single release note and tag for the entire monorepo.
+
+#### Commit Versioning
+[Conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) drive the version updates as follows:
+
+- A `fix` commit results in a patch bump (e.g., 1.0.0 to 1.0.1).
+- A `feat` commit results in a minor bump (e.g., 1.0.0 to 1.1.0).
+- The presence of a `BREAKING CHANGE:` in the commit footer leads to a major bump (e.g., 1.0.0 to 2.0.0).
```
Comment (jonathanpelletier) [inline README.md:64]: Just checking, will the release version have 3 digits for real?

**Review by jonathanpelletier — APPROVED** [review]
Looks good 👍 Got some questions though, but pre-approved.

File: README.md (hunk 2 above)
Comment (MPIngeno) [reply README.md:64]: With Lerna and the way it works with conventional commit versioning, yes 😢

**Review by JeanSebastienRoyIngeno — APPROVED** [review]
Haven't tried to apply, but the description seems clear to me

**Review by twistdmnd — APPROVED** [review]

**Pull request merged**